### PR TITLE
Feature/22 currently playing

### PR DESCRIPTION
**Pull request by shibeknight**
Issue #22 

Created a fluid card with a header for currently playing and added the following:

-An Item group that will hold games as Items using the CurrentItem component
-The CurrentItem component takes props that will display a game's title, platform (color icon) and description; it has a clickable rating component (WIP) and a conditional progress bar
- The Item group has a max height of 250px and is scrollable as more items are added

### CODE REVIEW

**Comment by shibeknight** [comment]
[Currently playing card component](https://trello.com/c/OEbkHxig/33-currently-playing-card-component)

**Review by LHBO19 — APPROVED** [review]

File: src/Components/CurrentItem.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Item, Rating, Icon, Progress } from 'semantic-ui-react';
```
Comment (avelinoschz) [inline src/Components/CurrentItem.js:3]: I guess Item is something more React-like, but how about naming as `Games` or everything related to this? You are already using words like Rating, Progress, maybe is a better fit for semantic purposes.

File: src/Components/CurrentlyCard.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Card, Item } from 'semantic-ui-react';
+import CurrentItem from './CurrentItem';
+
+const CurrentlyCard = () => {
```
Comment (avelinoschz) [inline src/Components/CurrentlyCard.js:5]: I'm not a big fan of super long names, but I think `CurrentlyCard` is not that much self explanatory, unless you know the context itself. We should avoid using React-ish names for more propietary variable names so we can distinguish better what is ours and what is more framework native.

File: src/Components/CurrentlyCard.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  return (
+    <Card raised fluid>
+      <Card.Content>
+        <Card.Header>My current list</Card.Header>
+        <Item.Group divided style={{ maxHeight: '250px', overflow: 'auto' }}>
+          <CurrentItem
```
Comment (avelinoschz) [inline src/Components/CurrentlyCard.js:11]: `CurrentItem` sounds also like selected or presented in this moment.

File: src/Components/CurrentItem.js (hunk 1 above)
Comment (shibeknight) [reply src/Components/CurrentItem.js:3]: In this case, Item is a component directly from Semant-UI so changing the name here is not an option; the same applies for Rating and the rest that are imported from semantic-ui-react

File: src/Components/CurrentlyCard.js (hunk 2 above)
Comment (shibeknight) [reply src/Components/CurrentlyCard.js:5]: I agree the naming of this component can change, right now it is just based on the base component used from semantic and the use for it as it is in the "currently playing" section.

File: src/Components/CurrentlyCard.js (hunk 3 above)
Comment (shibeknight) [reply src/Components/CurrentlyCard.js:11]: Same as above, naming should change to be more specific and self-explanatory as Item is already a predefined component from Semantic UI

File: src/Components/CurrentlyCard.js (hunk 2 above)
Comment (avelinoschz) [reply src/Components/CurrentlyCard.js:5]: Yes, I assumed it was for that. That's why i was thinking something like CurrentlyPlayindCard? Or something more creative haha

File: src/Components/CurrentlyCard.js (hunk 3 above)
Comment (avelinoschz) [reply src/Components/CurrentlyCard.js:11]: Yes, I think the same, only to be easier to distinguish them.

File: src/Components/CurrentItem.js (hunk 1 above)
Comment (avelinoschz) [reply src/Components/CurrentItem.js:3]: Oh yes, I didn't see Rating and Progress were imported from semantic-ui, I thought were custom. Ok, I understand.

**Review by avelinoschz — APPROVED** [review]

**Pull request merged**